src/utils/get-endpoints.ts

```diff
@@ -0,0 +1,15 @@
+import { HeliusCluster } from '../types'
+
+/**
+ * Retrieves the API and RPC URLs for the specified cluster
+ */
+export function getEndpoints(cluster: HeliusCluster): { api: string; rpc: string } {
+  switch (cluster) {
+    case "devnet":
+      return { api: "https://api-devnet.helius.xyz", rpc: "https://devnet.helius-rpc.com" }
```

Comment:
In https://github.com/helius-labs/helius-sdk/pull/31 the URLs of the RPC changed from `.helius.xyz` to `helius-rpc.com` and I was wondering if the API URLs have a similar counterpart or if they are good as-is?

Reply:
The API URLs are technically good as-is, however, some users get ad-blocked because of the `.xyz` domain so it's best to change them. The new URLs are `api-mainnet.helius-rpc.com` and `api-devnet.helius-rpc.com`